orthanc/orthanc-raw/README.md

```diff
@@ -43,7 +43,7 @@ secrets. Orthanc interprets all `.json` files in the `/run/secrets` mount as con
 
 ### Step 1
 
-Save credentials `.env` for the PACS/VNA Q/R target, postgreSQL and 'Orthanc anon'.
+Save credentials `.env` for the VNA (primary) and PACS (secondary) Q/R targets, postgreSQL and 'Orthanc anon'.
```

Comment:
might be worth saying that the secondary is optional, and if it is not set then only the primary will be used?